openshift/patches/023-configmap-trusted-cabundle.patch

```diff
@@ -28,4 +28,6 @@ index 000000000..a4c1a5f73
 +    app.kubernetes.io/name: knative-eventing
```

Comment:
is this adding different formatting? :thinking:

Reply:
I think this looks always that way (important are the lines with color on the line number). See e.g. https://github.com/openshift-knative/eventing/pull/513/files#diff-ef7ff98cd21face0992b9799f9242ce644b7674fb85d22a64ec0ec79c0484d43

Reply:
@matzew can you recheck?

Reply:
I still see lots of exact - instead of ust that annotation :thinking:

Reply:
@matzew you are considering that this is a git patch and GH does render everything as change, right?